### ISSUE TITLE
Link the filetypes and colorthemes

### ISSUE THREAD

**Issue by XVilka** [vifm/vifm#453]
So instead of repeating matches like
```vim
highlight /^.*\.(zip|gz|bz2|xz|tar|tgz|tbz2|7z|rar|iso|rpm|deb)$/ ctermfg=red
```
It would be possible to define a link, like in vim:
```
filextype myimage *.jpg,*.jpeg,*.png,*.gif
       \ {View in feh}
       \ feh -FZ %d --start-at %d/%c 2>/dev/null
hi link mymage Image
```
where `Image` is defined in the colorscheme
```vim
highlight Image cterm=bold ctermbg=black ctermfg=red
```
It would simplify creating the proper file type coloring, and handling

**Comment by xaizek** [comment]
Something like this would be useful. It was suggested before to reduce duplication between `:filetype` and `:fileviewer`, highlighting couldn't be file specific back then. Maybe it can all be solved in one go (e.g., by decoupling group definition from associations).

**Comment by XVilka** [comment]
@xaizek indeed, decoupling group definition from everything would be even better.
